src/common/components/elements/NFTPreviewGrid.tsx

```diff
@@ -73,6 +74,10 @@ const getOverlayStatus = (index: number, nftValues?: NFTValue[]) => {
   );
 };
 
+const StyledAntDImage = styled(AntImage)`
```

Comment:
Replaced with AntD image to get the automatic Preview functionality. Ref: https://holaplex.atlassian.net/jira/software/projects/HOLA/boards/2?selectedIssue=HOLA-72